coderd/httpmw/organizationparam.go

```diff
@@ -180,7 +180,7 @@ func ExtractOrganizationMember(ctx context.Context, auth func(r *http.Request, a
 	organizationMembers, err := db.OrganizationMembers(ctx, database.OrganizationMembersParams{
 		OrganizationID: orgID,
 		UserID:         user.ID,
-		IncludeSystem:  false,
+		IncludeSystem:  true,
```

Comment:
review: this is required in order to be able to delete a prebuilt workspace from the CLI, as it needs to fetch it first.